_maps/map_files/NSSJourney/NSSJourney.dmm

```diff
@@ -53821,6 +53821,16 @@
 	},
 /turf/open/floor/iron,
 /area/engineering/atmos)
+"udf" = (
+/obj/structure/rack,
+/obj/item/gun/ballistic/rocketlauncher/secuirty,
+/obj/item/gun/ballistic/rocketlauncher/secuirty,
+/obj/item/gun/ballistic/rocketlauncher/secuirty,
+/obj/item/storage/belt/military/rocket/full,
+/obj/item/storage/belt/military/rocket/full,
```

Comment:
Don't give them three launchers and 3 belts ffs. Give them 1

Reply:
Given that you can only print more ammo for it, and not more weapons.
I might go ahead and just knock it down to two starting instead.
Would make it one, but then I would want to add em to cargo.